python/cudf/cudf/core/multiindex.py

```diff
@@ -1107,29 +1104,6 @@ def get_level_values(self, level):
         )
         return level_values
 
-    def _to_frame(self):
```

Comment:
Where did this move to? Is it just not needed/used anymore?

Reply:
Yeah just removed. Previously, there was a `_to_frame` method defined in `index.py` that was called by `Index.to_frame` and `GenericIndex.to_frame`, but the methods were completely identical so I got rid of `_to_frame` and just inlined it in the new `SingleColumnFrame` class. In the process, I realized that `multiindex.py` copied the `_to_frame` method over, but never actually used its implementation anywhere.